app/src/main/java/org/mifos/selfserviceapp/ui/fragments/HomeFragment.java

```diff
@@ -127,6 +129,13 @@ public void showUserImage(final Bitmap bitmap) {
             public void run() {
                 ((ImageView) toolbarView.findViewById(R.id.iv_user_image)).
                         setImageBitmap(bitmap);
+                toolbarView.findViewById(R.id.iv_user_image).
+                        setOnClickListener(new View.OnClickListener() {
```

Comment:
@therajanmaurya as toolbar custom view is added in onResume and removed at onPause state https://github.com/dilpreet96/self-service-app/blob/abb8736564ef83d009489ce19c05234953c7da88/app/src/main/java/org/mifos/selfserviceapp/ui/fragments/HomeFragment.java#L207
so, butterknife is not be able to find the image as it is added at runtime due to which I used the manual method